app.js

```diff
@@ -199,10 +199,15 @@ define(function(require) {
 							header: true,
 							skipEmptyLines: true,
 							complete: function(results) {
-								var fileColumns = results.meta.fields,
+								var fileColumns = _.map(results.meta.fields, _.trim),
 									formattedData = {
 										fileName: file.name,
-										records: results.data,
+										records: _.map(results.data, function(record) {
+											_.mapKeys(record, function(value, key) {
+												return _.trim(key);
+											});
+											return record;
```

Comment:
Not sure this has any effects.

Considering that `lodash#mapKeys` returns a new object, shouldn't the map iterating function return that result rather than the unchanged `record` object?